Assignment1/src/main/java/org/christmas/lights/grid/ChristmasLightGrid.java

```diff
@@ -0,0 +1,10 @@
+package org.christmas.lights.grid;
+
+public interface ChristmasLightGrid {
+
+	
+	void switchOnLight(int row, int column);
```

Comment:
toggle and getLightOnCount can be put into separate interfaces and this violates interface segregation and single responsibility principle

toggle is not unique to a light

Reply:
Moved as suggested